src/common/BaseAnnotator.ts

```diff
@@ -103,13 +103,12 @@ export default class BaseAnnotator extends EventEmitter {
     };
 
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    public scrollToAnnotation(annotationId: string | null): void {
+    public scrollToAnnotation(annotation: Annotation): void {
```

Comment:
How about `public abstract scrollToAnnotation(annotation: Annotation): void;`

Reply:
This appears to be a breaking change to an existing API in [2.3.0](https://github.com/box/box-annotations/blob/v2.3.0/src/Annotator.js#L617) and [3.13.0](https://github.com/box/box-annotations/blob/v3.13.0/src/Annotator.js#L493).